open_bounties/0090 Interactive Tutorial for Send a Cross-Currency Payment.md

```diff
@@ -0,0 +1,41 @@
+---
+ID: "0090"
+Discussion: https://github.com/XRPLBounties/Proposals/discussions/0090
+Category: Interactive Tutorials 
+Status: Open
+Amount: $2,000
+---
+
+
+# Interactive Tutorial for Send a Cross-Currency Payment
+Bounty proposals require 3+ technical approvals _and_ budget approval before they are available to be worked on. A proposal PR with 3+ ✅s is not fully approved if it has not been merged. The bounty is only ready to be worked on after the PR has been merged.
+
+## Overview
+
+### High Level Description
+Build an interactive tutorial for sending a cross-currency payment on xrpl.org, reference issues [here](https://github.com/XRPLF/xrpl-dev-portal/issues/1198)
+
+### requirements
+1. Write quality code samples in Javascript that shows steps to send a cross-currency payment, including: 
```

Comment:
```suggestion
1. Write quality code samples in Javascript that show steps to send a cross-currency payment, including: 
```

nit: I think it should be show instead of shows right? My mind is blanking right now on English. 😂